mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go

```diff
@@ -261,7 +276,9 @@ func (p EBSPlugin) FetchMetrics() (map[string]interface{}, error) {
 		volumeID := normalizeVolumeID(*vol.VolumeId)
 		graphs := defaultGraphs
 		if *vol.VolumeType == "io1" {
-			graphs = allGraphs
+			graphs = io1Graphs
+		} else if *vol.VolumeType == "gp2" {
```

Comment:
According to the Cloudwatch reference, BurstBalance metric is also available on st1 and sc1 volumes.
Please support them also.